### ISSUE TITLE
Next: a11y review

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#564]
Following a backlog of the a11y review I just finish with Fabri. 

### Fixes to do ASAP:
- [x] add an `h2` title to the **footer** <<< ✅ fixed here https://github.com/italia/designers.italia.it/commit/de53b0fac371e7c4e39fed86e5f2663c602f4d45
- [x] no primary color for **disabled/coming soon links** on the main menu, best to add something like (presto) as a coming soon label <<<< ✅ fixed here https://github.com/italia/designers.italia.it/commit/37ea0cfea456f6b6857638ad1d1a6446aa6a879b
- [x] **share links:** add a visually hidden ` (si apre in una nuova finestra)` for SR <<< ✅ fixed here https://github.com/italia/designers.italia.it/commit/bb4852ad929724eaac2350077ff194d6f3865245 (backlog: we must consider to move sharers' data to their src/data/share.yaml setting file as soon as possible @bfabio ) 
- [x] 🟡 **free input text field (feedback)** must have an `id` on the helper text below (max 200 chars) and an `ariadescribedby` that point to that id on the input field *(do we need to fix this also on Bootstrap Italia @astagi ?) <<< ✅ fixed with #566 
- [x] **the design system sidebar nav** should be outside the `main` region... a first quick fix could be to add specific skip links for these pages <<<< ✅ fixed by @gpeirolo here https://github.com/italia/designers.italia.it/pull/565/
- [x] **component viewer:** add visually hidden start and finish to the component preview <<< ✅ fixed here https://github.com/italia/designers.italia.it/commit/60f3e25d45bd4b4502dfe747874df42b771d20c2
- [x] **component viewer:** hide aria-hidden the wrap option, not useful for SR <<< ✅ fixed here https://github.com/italia/designers.italia.it/commit/023196c613b8ac35451b8713080fcfa33c2c6b92
- [ ] <<<< ♻️This has been converted to dedicated issue #572 >>>
- [x] **chips "Argomento":** remove the visually hidden helper on each one... and use instead a visually hidden unique title for the areas where we show them. <<< ✅ fixed here https://github.com/italia/designers.italia.it/commit/e607961e1027effc1f4375dc4bb2f1df5549fb64
- [x] **Header-pre** must be before skiplinks <<< ✅ fixed here https://github.com/italia/designers.italia.it/pull/569
- [x] **Header-post** should be in a dedicated section between the header and main, not in the main. We need also to render it as a list and not as a nav. <<< ✅ fixed here https://github.com/italia/designers.italia.it/pull/570
- [ ] <<<< ♻️This has been converted to dedicated issue #571 >>>

### Future automation we would need: 
1. 🟠 recognize and address external links in markdown fields (medium priority) <<< ♻️ We have already an issue for this here https://github.com/italia/designers.italia.it/issues/543 >>> 
2. 🟡 assign unique ids to component preview imported from Bootstrap Italia (no priority) <<< ♻️ We have already an issue for this here https://github.com/italia/designers.italia.it/issues/535 >>>

### To evaluate: 
1. ❓the feedback title should be the legend of the fieldset sì/no? <<< ❓waiting a feedback from Fabri. I will open a dedicated issue if needed later.

**Comment by bfabio** [comment]
>  orange_circle toast (copy actions for share and component viewer): in an ideal world a toast like ours should be implemented as an empty and hidden div with aria-live="polite" always present on the page, filled at the right moment with the right text via js.

I came to the same conclusion when trying to implement it in the link sharing widget. This should be part and come out of the box in our [opinionated framework for building sites](https://github.com/teamdigitale/gatsby-site)

**Comment by Fupete** [comment]
Update: I just open an issue https://github.com/italia/bootstrap-italia/issues/789 on Bootstrap Italia for the 🔴 Tab issue.

**Comment by Fupete** [comment]
Ok, fixed almost everything with the help of @gpeirolo for a couple. 
@astagi @bfabio I move the Toast and Tab ones on dedicated issues to track them. 

I think we can close this main review at this stage.
